### ISSUE TITLE
[DD] Initial FE implementation 

### ISSUE THREAD

**Issue by Dr-Pongo** [department-of-veterans-affairs/va.gov-team#97002]
## Issue Description
Create an initial version of the Digital Dispute with all required pages. We don't have a URL yet, or a fully functioning vets-api endpoint, so this can live on a remote branch until we're ready to merge. 

- Backend ticket for prefill stuffs:
department-of-veterans-affairs/va.gov-team/issues/98021

### Definitions:
- POST: `debts_api/v0/digital_disputes`
- App name in `vets-website`: `src/applications/dispute-debt`
- Root URL: `<env>/manage-va-debt/dispute-debt/`

### Links:
- **[Figma page layouts](https://www.figma.com/design/D8tfoPhUZlqBUomMTxzkQb/Dispute-Form-(VHA-VBA)?node-id=19-220&node-type=canvas&t=oApc5J3nesSo7nLx-0)**
- **[Application generator](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-gov-application-generator)**
- **[CAIA Mural with url definitions and other helpful stuffs](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1731461600152/93a4a19b003f01b86534471cb686f81bf073e4a2)**
- **[URL Guidance](https://design.va.gov/components/url-standards/)**
- **[VA Forms Library](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-getting-started-with-the-forms-li)**

## Tasks
- [x] Implement first version of the form for vets-website
- [ ] Include necessary unit tests
- [ ] Once initial draft is up, review with Design

## Design and Front End sync on implementation of design
- [x] Required
- [ ] Complete
- [ ] Not Required           

## Acceptance Criteria
- [ ] Initial version of Digital dispute is created
- [ ] Initial version has been reviewed with design

## Testing
- [ ] N/A if non development work
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description     

Test procedures:      
*If additional testing steps or credentials to perform testing are needed list them here*    

## Ticket Creation - Definition of Ready
- [x] Description written
- [x] Tasks defined
- [x] Acceptance criteria written
- [x] Testing defined
- [x] Add Labels
- [x] Attach to an Epic
- [x] Estimate Points

**Comment by Dr-Pongo** [comment]
Rolling over because it's a big effort that got started late in the sprint!

**Comment by Dr-Pongo** [comment]
Feature flag is nearly ready to be merged
- [DD] Create Digital Dispute feature toggle [#96916](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/96916)

**Initial PR for Digital Dispute in `vets-website`**
- department-of-veterans-affairs/vets-website/pull/33151 

**Initial PR for DIgital DIspute in `vets-api`**
- department-of-veterans-affairs/vets-api/pull/19545
[Todo list after initial PR is merged](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/benefits-memorials-2/engineering/general/digital-dispute/dev-notes.md#todo---after-initial-commit)

**Comment by Dr-Pongo** [comment]
Opened PR for vets-json-schema which is the true first step in merging on vet-website and vets-api. Waiting on feedback on PRs before proceeding and discussing structure and future steps/work the schema in general and how it aligns with the white list for api.

**Comment by Dr-Pongo** [comment]
After some changes suggested by the team, I opened a support ticket to get a platform review!
https://dsva.slack.com/archives/CBU0KDSB1/p1733156945943879

**Comment by Dr-Pongo** [comment]
Need to address some of the CI issues for the vets-website to get it to merge (close PR, rebase, open new PR?)
. We can likely close the ticket after that, although the BE PR may block it as well.

**Comment by Dr-Pongo** [comment]
So fresh so clean new FE PR (rebased and made new PR to hopefully pickup the schema changes
- department-of-veterans-affairs/vets-website/pull/33332

**Comment by Dr-Pongo** [comment]
Rolled into 140 while troubleshooting errors and managing other high priority tasks

**Comment by Dr-Pongo** [comment]
Still troubleshooting CI/CD issues

**Comment by Dr-Pongo** [comment]
Opened support ticket looking for assistance with CI issues:
**https://dsva.slack.com/archives/CBU0KDSB1/p1733761378310059**

**Comment by Dr-Pongo** [comment]
Supporting PRs got merged to fix some platform level issues, waiting for platform approval. If we don't get it by end of sprint, I'll likely close the PR, and leave notes in FE documentation on the branch that contains the skeleton for Dispute Debt so we can hit the ground running in the new contract

**Comment by Dr-Pongo** [comment]
Closing out at end of sprint/contract. The initial app implementation is in the remote branch `dd/97002/new-app-initial`, and we'll still need a content-build PR to update the registry with the new URL.
